src/cmstest/main.c

```diff
@@ -228,82 +230,85 @@ int main(int argc __attribute__((unused)), char *arg[] __attribute__((unused)))
-        goto end;
-      }
+        output_info = XRRGetOutputInfo(display, rsrc, crtc_info->outputs[output]);
+        if(!output_info)
+        {
+          printf("can't get output info for screen %d CRTC %d output %d\n", screen, crtc, output);
+          goto end;
+        }
 
-      if(output_info->connection == RR_Disconnected)
-      {
-        printf("screen %d CRTC %d output %d is disconnected, skipping\n", screen, crtc, output);
-        goto end;
-      }
+        if(output_info->connection == RR_Disconnected)
+        {
+          printf("screen %d CRTC %d output %d is disconnected, skipping\n", screen, crtc, output);
+          goto end;
+        }
 
-      monitor_t *monitor = (monitor_t *)calloc(1, sizeof(monitor_t));
+        monitor_t *monitor = (monitor_t *)calloc(1, sizeof(monitor_t));
 #if 0
-      // in case we also want the edid data
-      Atom edid_atom = XInternAtom(display, "EDID", False), actual_type;
-      int actual_format;
-      unsigned long nitems, bytes_after;
-      unsigned char *prop;
-      int res = XRRGetOutputProperty(display, rsrc->outputs[output], edid_atom, 0, G_MAXLONG, FALSE,
-      FALSE, XA_INTEGER, &actual_type, &actual_format, &nitems, &bytes_after, &prop);
-      if(res == Success && actual_type == XA_INTEGER && actual_format == 8 && nitems != 0)
-      {
-        printf("EDID for %s has size %lu\n", output_info->name, nitems);
-        // TODO: parse the edid blob in prop. since that is really ugly code I left it out for now.
-      }
-      if(prop)
-        XFree(prop);
+        // in case we also want the edid data
+        Atom edid_atom = XInternAtom(display, "EDID", False), actual_type;
+        int actual_format;
+        unsigned long nitems, bytes_after;
+        unsigned char *prop;
+        int res = XRRGetOutputProperty(display, rsrc->outputs[output], edid_atom, 0, G_MAXLONG, FALSE,
+        FALSE, XA_INTEGER, &actual_type, &actual_format, &nitems, &bytes_after, &prop);
+        if(res == Success && actual_type == XA_INTEGER && actual_format == 8 && nitems != 0)
+        {
+          printf("EDID for %s has size %lu\n", output_info->name, nitems);
+          // TODO: parse the edid blob in prop. since that is really ugly code I left it out for now.
+        }
+        if(prop)
+          XFree(prop);
 #endif
 
-      monitor->root = root;
-      monitor->screen = screen;
-      monitor->crtc = crtc;
-      monitor->is_primary = is_primary;
-      monitor->atom_id = atom_id++;
-      monitor->name = g_strdup(output_info->name);
-      monitor_list = g_list_append(monitor_list, monitor);
-
-end:
-      XRRFreeCrtcInfo(crtc_info);
-      XRRFreeOutputInfo(output_info);
+        monitor->root = root;
+        monitor->screen = screen;
+        monitor->crtc = crtc;
+        monitor->is_primary = is_primary;
+        monitor->atom_id = atom_id++;
+        monitor->name = g_strdup(output_info->name);
+        monitor_list = g_list_append(monitor_list, monitor);
+
+      end:
```

Comment:
LebedevRI insisted on clang-format... and these are side effects (w/ clang-format 3.8)